addons/mass_mailing/wizard/mailing_list_add_contact.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailingListAddContact(models.TransientModel):
+    _name = "mailing.list.add.contact"
+    _description = "Add contact to mailing list wizard"
```

Comment:
Don't think specifying that it's a wizard is useful in the description
> Add contacts to mailing list

Should be enough.